docs/standard/index.md

```diff
@@ -1,60 +1,60 @@
 ms.technology: dotnet-standard
 ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 ---
-# .NET Guide
+# .NET Standard guide
 
-The .NET Guide provides a large amount of information about .NET.  Depending on your familiarity with .NET, you may wish to explore different sections of this guide and other sections of the .NET documentation.
+The .NET Standard guide provides information about .NET Standard. Depending on your familiarity with .NET, you may wish to explore different sections of this guide and other guides such as [.NET Core](../core/index.md) or [.NET Framework](../framework/index.md).
 
 ## New to .NET
 
-If you want a high-level overview about .NET, check out [What is .NET?](https://dotnet.microsoft.com/learn/dotnet/what-is-dotnet).
+If you want a high-level overview of .NET, see [What is .NET?](https://dotnet.microsoft.com/learn/dotnet/what-is-dotnet).
 
-If you're new to .NET, check out the [Get Started](get-started.md) article.
+If you're new to .NET, see [Get started](get-started.md).
 
-If you prefer to have a guided tour through major features of .NET, check out the [Tour of .NET](tour.md).
+If you prefer to have a guided tour through major features of .NET, see [Tour of .NET](tour.md).
 
-You can also read about [.NET Architectural Components](components.md) to get an overview of the various "pieces" of .NET and how they fit together.
+For an overview of the various "pieces" of .NET and how they fit together, see [.NET architectural components](components.md).
 
 ## New to .NET Core
 
-If you're new to .NET Core, check out [Get Started with .NET Core](../core/get-started.md).
+If you're new to .NET Core, see [Get started with .NET Core](../core/get-started.md).
 
 ## New to .NET Standard
 
-If you're new to .NET Standard, check out [.NET Standard](net-standard.md).
+If you're new to .NET Standard, see [.NET Standard](net-standard.md).
 
-## Porting .NET Framework Code to .NET Core
+## Port .NET Framework code to .NET Core
 
-If you're looking to port an application, service, or some component of a system to .NET Core, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
+If you want to port an application, service, or some component of a system to .NET Core, see [Port to .NET Core from .NET Framework](../core/porting/index.md).
 
-## Porting a NuGet package from .NET Framework to .NET Standard or .NET Core
+## Port a NuGet package from .NET Framework to .NET Standard or .NET Core
 
-If you're looking to port a NuGet package to .NET Standard, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).  Tooling for .NET Standard and .NET Core are shared, so the content will be relevant for porting to .NET Standard as well as .NET Core.
+If you want to port a NuGet package to .NET Standard, see [Port to .NET Core from .NET Framework](../core/porting/index.md). Tooling for .NET Standard and .NET Core is shared, so the content is relevant for porting to both .NET Standard and .NET Core.
```

Comment:
This section links to the same article as the previous one. Maybe the previous one can be deleted ?